node/src/server-modules/GlideJson.ts

```diff
@@ -308,4 +308,49 @@ export class GlideJson {
 
         return _executeCommand<number>(client, args);
     }
+
+    /**
+     * Reports the type of values at the given path.
+     *
+     * @param client - The client to execute the command.
+     * @param key - The key of the JSON document.
+     * @param options - (Optional) Additional parameters:
+     * - (Optional) path - If `null`, deletes the entire JSON document at `key`.
+     * @returns ReturnTypeJson:
+     *   - If one path is given:
```

Comment:
I believe this command can only accept one single path, so we don't need this line.